### DISCUSSION TITLE
图片的地址和图片链接有什么区别，这难道不是一个东西吗

### DISCUSSION THREAD

**Discussion by ChenKun1997** [wangeditor-team/wangEditor#4367] [General]
![image](https://user-images.githubusercontent.com/24249578/173476122-5645b8e8-5474-468c-824e-0790caedea06.png)
如图，我试了一下只有传图片的地址才有用，想问下图片的链接是用来做什么的，可有可无

**Comment by wangfupeng1988** [comment]
图片地址是 src

图片链接，是为了在显示图片时，点击图片跳转到其他的网页
